src/directives/index.ts

```diff
@@ -0,0 +1,14 @@
+export { Directive } from "./main"
+export { admonitions } from "./admonitions"
+export { code } from "./code"
+export { images } from "./images"
+export { default as directivePlugin } from "./plugin"
+export type { IOptions as IDirectiveOptions } from "./plugin"
+export { tables } from "./tables"
+
+import { admonitions } from "./admonitions"
+import { code } from "./code"
+import { images } from "./images"
+import { tables } from "./tables"
+
+export const directives = { ...admonitions, ...images, ...code, ...tables }
```

Comment:
nitpick: maybe `directivesDefault` / `rolesDefault` would be more descriptive

Reply:
will update.